### PR TITLE
Update create-a-keyword-dictionary.md

### PR DESCRIPTION
**Pull request by msbemba**
It supports all languages. Added the same to the document (this has already been updated in another document - https://docs.microsoft.com/en-us/microsoft-365/compliance/communication-compliance-feature-reference?view=o365-worldwide)

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2436

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/create-a-keyword-dictionary.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ description: "Learn the basic steps to creating a keyword dictionary in the Offi
...
-  
+Data loss prevention (DLP) can identify, monitor, and protect your sensitive information. Identifying sensitive information sometimes requires looking for keywords, particularly when identifying generic content (such as healthcare-related communication), or inappropriate or explicit language. Although you can create keyword lists in sensitive information types, keyword lists are limited in size and require modifying XML to create or edit them. Keyword dictionaries provide simpler management of keywords and at a much larger scale, supporting up to 100,000 terms per dictionary and and support any language.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-a-keyword-dictionary.md]:
```suggestion
Data loss prevention (DLP) can identify, monitor, and protect your sensitive information. Identifying sensitive information sometimes requires looking for keywords, particularly when identifying generic content (such as healthcare-related communication), or inappropriate or explicit language. Although you can create keyword lists in sensitive information types, keyword lists are limited in size and require modifying XML to create or edit them. Keyword dictionaries provide simpler management of keywords and at a much larger scale, supporting up to 100,000 terms per dictionary and support any language.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks.

**Pull request merged**